indexer/packages/v4-protos/src/codegen/klyraprotocol/yields/genesis.ts

```
@@ -0,0 +1,38 @@
+import * as _m0 from "protobufjs/minimal";
+import { DeepPartial } from "../../helpers";
+export interface GenesisState {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)